vtexdocs/devportal
# feat(release-note): toggle and months

#### What is the purpose of this pull request?

Add the missing fields, implement the toggle and split releases by months and years.

[Docs here](https://www.notion.so/vtexhandbook/Adicionar-campos-criar-toggle-e-seccionar-por-meses-f7f9fbe473d642798ed88122956ee076) 📖 

#### What problem is this solving?

- Some fields that are in release format in the content repository were not implemented yet. The slug and hidden. They were added and implemented according to their expected functionality.

- A toggle was needed to open and close the release, thus showing the description of the update. The first release should be open after rendering and all others closed.

- Based on the date of the releases, it was necessary to group the releases according to month and year.

#### How should this be manually tested?

Please go to [this workspace,](https://deploy-preview-42--elated-hoover-5c29bf.netlify.app/) and check that everything works correctly. Also compare it with the version in the [figma](https://www.figma.com/file/Lx6sXdrw9KEvtSEKWttmv8/Developer-Portal?node-id=129%3A28).

#### Screenshots or example usage

<img width="748" alt="image" src="https://user-images.githubusercontent.com/32322512/171724691-509b18dd-aa60-41d4-96f1-36ea3c6bc931.png">


#### Types of changes
